source/llassetgen-rendering/main.cpp

```diff
@@ -179,7 +205,73 @@ class Window : public WindowQt {
         doneCurrent();
     }
 
-    void applyColorChange(float &color, QString value) {
+    virtual void mousePressEvent(QMouseEvent* event) override {
+        makeCurrent();
+
+        lastMousePos.x = event->x();
+        lastMousePos.y = event->y();
+
+        if (event->button() == Qt::LeftButton) {
+            isPanning = true;
+        } else if (event->button() == Qt::RightButton) {
+            isRotating = true;
+        }
+
+        doneCurrent();
```

Comment:
We don't need it in `mouseReleaseEvent` because there we do nothing with OpenGL.
I just had a further look:
(1) I remember that I wrote those calls in mousePressEvent because I had OpenGL-functions there. Since they are gone now, I can delete those calls.
(2) We actually don't need it here in `mouseMoveEvent` because in `paint()` we also have those calls. 

I'll fix this.
Thanks!